lib/polyfill/media_capabilities.js

```diff
@@ -54,15 +54,17 @@ shaka.polyfill.MediaCapabilities = class {
     // Use 'MediaSource.isTypeSupported' to check if the stream is supported.
     if (mediaDecodingConfig['video']) {
       const contentType = mediaDecodingConfig['video'].contentType;
-      const isSupported = MediaSource.isTypeSupported(contentType);
+      const isSupported =
+          window.MediaSource && MediaSource.isTypeSupported(contentType);
```

Comment:
Ah, I see.  Could this occur on iOS, since there's no MediaSource there?

Although this would fix an exception being thrown when MediaSource is not available, your solution would call all content unsupported on such a platform.  Why would that be okay?